examples/3d-tiles/webpack.config.js

```diff
@@ -9,6 +9,10 @@ const COMMON_CONFIG = {
     library: 'App'
   },
 
+  resolve: {
+    mainFields: ['esnext', 'browser', 'module', 'main']
```

Comment:
what is `mainFields` used for?

Reply:
Choosing dist. 

https://webpack.js.org/configuration/resolve/#resolvemainfields

The idea is to choose the `es6` dist, which should be very lightly transpiled. In particular, it should not transpile async using generators.